python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -263,6 +294,58 @@ def f():
     assert ray.get(f.remote())
 
 
+@pytest.mark.skipif(
+    os.environ.get("CI") is None, reason="This test is only run on CI.")
+@pytest.mark.skipif(
+    sys.platform != "linux", reason="This test is only run for Linux.")
```

Comment:
why?

Reply:
I just added this because it is how these other tests are marked.

@architkulkarni is there a reason for this?

Reply:
I think this can only be run on buildkite, because it needs to be run in a special job after the wheel is built (because the tests insert the wheel for the current build into the runtime_env).  And right now only Linux is running on buildkite.  @simon-mo knows more about this

Reply:
I can add a better `reason=` (basically explaining what I said above) in a future PR